cmd/nameset-backfill/main.go

```diff
@@ -71,18 +71,18 @@ func (b *backfiller) run() error {
 
 func (b *backfiller) findCerts() ([]resultHolder, error) {
 	var allResults []resultHolder
+	lastSerial := ""
 	for {
 		var results []resultHolder
 		_, err := b.dbMap.Select(
 			&results,
 			`SELECT c.serial, c.issued, c.expires, c.der FROM certificates AS c
        LEFT JOIN fqdnSets AS ns ON c.serial=ns.serial
        WHERE ns.serial IS NULL
-       ORDER BY c.issued DESC
-       LIMIT ?
-       OFFSET ?`,
+       AND c.serial > ?
```

Comment:
We don't have a auto-increment column, the primary key of `certificates` is `serial` which is `VARCHAR(255)`. This means that when stored the natural ordering of the index is lexicographic, i.e. `SELECT * FROM certificates LIMIT 10;` won't return the first 10 by insertion order but by the alphabetical ordering of the serials, which we are taking advantage of here.